src/Components/addItem.js

```diff
@@ -1,40 +1,91 @@
 import React from 'react';
+import '../App.css';
 import { fb } from '../lib/firebase';
 
 class AddItem extends React.Component {
   constructor() {
     super();
     this.state = {
       itemName: '',
+      purchaseFrequency: null,
+      lastPurchasedDate: null,
     };
   }
   updateInput = e => {
     this.setState({
       [e.target.name]: e.target.value,
     });
   };
+
+  updateFrequency = e => {
+    this.setState({
+      purchaseFrequency: e.target.value,
```

Comment:
Great update! 👏🏽 I now see what I was suggesting wasn't even in the right place. Love how you solved this.